topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md

```diff
@@ -0,0 +1,444 @@
+>    - *"name of the embedding to plot"*: `draw_graph_fa`
+>    - *"color by attributes, comma separated texts"*: `cell_type`
+>    - *"Use raw attributes if present"*: `No`
+>    - *"Location of legend"*: `On data`
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. Does this plot seem better or worse than before? Remember that we're trying to understand the relationships between our groups of cells in time. 
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Oh dear! This doesn't look great. Maybe the DP-M4 cells are a whole other trajectory? That doesn't seem right. Saying that, this spreads out our T-mature cells, which makes a lot more sense when it comes to T-cell biology (we expect T-cells to differentiate into two types of T-cells, Cd8+Cd4- and Cd4+Cd8-). If you wanted to, you could also re-cluster your cells (since you've changed the neighborhood graph on which the clusterisation depends). However, we tried that, and it called far too many clusters given the depth of sequencing in this dataset. Let's stick with our known cell types and move from there.
+> >
+> > ![The T mature and DP L clusters have been stretched out](../../images/scrna-case_trajectories/TrajectoriesFDG2.png "FDG Plot after recalculating neighbours from the diffusion map")
+> {: .solution}
+>
+{: .question}
+
+## Working in a group? Decision-time!
+If you are working in a group, you can now divide up a decision here with one *control* and the rest can vary numbers so that you can compare results throughout the tutorials.
+- Control
+   - Go straight to the [PAGA section](#paga)
+- Everyone else:
+   - you could recluster your cells using {% tool [Scanpy FindCluster](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_find_cluster/scanpy_find_cluster/1.8.1+galaxy0) %} at a different resolution, perhaps lower than the 0.6 we used before (Take a look at the Cell clusters step in the [Filter, Plot and Explore]({% link topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.md %}) tutorial if you need help with this.
+        - Please note that in this case, you will want to change the PAGA step `sc.pl.paga` to group by `louvain` rather than `cell_type`. You can certainly still plot both, we only didn't because with using our old Louvain calls, the cell_type and louvain categories are identical.
+   - you could undo the optional diffusion map step by recalculating the neighbours again using `X_pca` instead of `X_diffmap`
+   - you could also try changing the number of neighbors used in that step 
+
+- Everyone else: You will want to compare FREQUENTLY with your control team member.
+
+## PAGA
+
+[PAGA](https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.paga.html) is used to generalise relationships between groups, or likely clusters, in this case. It will make it much easier to see the trajectories between our clusters of T cells. 
```

Comment:
```suggestion
[PAGA](https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.paga.html) is used to generalise relationships between groups, or likely clusters, in this case. It will make it much easier to see the trajectories between our clusters of T-cells. 
```